settings.py

```diff
@@ -496,6 +496,9 @@
     ]
 }
 
+# Note: if you change this, make sure to update the cronjob in commcarehq-ansible
+RESTORE_PAYLOAD_DIR = os.path.join(FILEPATH, 'restore_payloads')
```

Comment:
This makes sense to me. `restore_payloads` in the root dir felt wrong to me as well, but i didn't know where else to put it. Looking into making this change